app/services/[serviceId]/users/[userId]/_components/case-records-cards.tsx

```
@@ -0,0 +1,442 @@
+  // A.T専用の暫定止血（Console 500連打回避）
+  // TODO: 根本解決後にこのガードを削除
+  if (userId === AT_USER_ID) {
+    return null
+  }
+
+  useEffect(() => {
+    setForm(defaultForm(userId, serviceId))
+  }, [userId, serviceId])
+
+  useEffect(() => {
+    // didFetchRef で初回のみ実行制御（userId/serviceId変更時は再度実行したいため、ここでリセット不要）
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)